### PR TITLE
Update ibc_data_schema and community guidelines

### PR DESCRIPTION
**Pull request by shahbazn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,76 @@
...
+### Steps to create new IBC paths
 
 We recommend using hermes relayer implementation and use this [tutorial](https://hermes.informal.systems/tutorials/local-chains/add-a-new-relay-path.html) to create new paths.
 
-# Requesting a new IBC path
+## Requesting a new IBC path
+
+If you are builder on Archway protocol and/or looking to connect Archway Networks to another chain through IBC, but do have the expertise or the capacity to operate relayers, you can request the desired IBC relayer from archway community:
+
+- Join [Arcway Discord](https://discord.gg/5FVvx3WGfa)
```
Comment (kayano) [inline README.md:24]: `Arcway` - typo

File: README.md [hunk 2]
```diff
@@ -1,26 +1,76 @@
...
 
-## Steps to create new IBC paths
+### Steps to create new IBC paths
 
 We recommend using hermes relayer implementation and use this [tutorial](https://hermes.informal.systems/tutorials/local-chains/add-a-new-relay-path.html) to create new paths.
 
-# Requesting a new IBC path
+## Requesting a new IBC path
+
+If you are builder on Archway protocol and/or looking to connect Archway Networks to another chain through IBC, but do have the expertise or the capacity to operate relayers, you can request the desired IBC relayer from archway community:
```
Comment (emperorjm) [inline README.md:22]:
```suggestion
If you are a builder on the Archway protocol and/or looking to connect Archway Networks to another chain through IBC, but do not have the expertise or the capacity to operate relayers, you can request the desired IBC relayer from archway community:
```

File: ibc_data_schema.json [hunk 3]
```diff
@@ -80,7 +80,43 @@
...
+            "type": "object",
+            "$ref": "#/$defs/operator_info"
+          },
+          "chain_2": {
+            "type": "object",
+            "$ref": "#/$defs/operator_info"
+          },
+          "description": {
+            "type": "string",
+            "description": "Relayer addresses and metadata."
```
Comment (kayano) [inline ibc_data_schema.json:103]: `description` property should contain `Relayer addresses and metadata`? Can you give an example value?

File: README.md [hunk 4]
```diff
@@ -1,26 +1,76 @@
...
+
+- Join [Arcway Discord](https://discord.gg/5FVvx3WGfa)
+- Use "Mainnet Validators" or "Relayers" tag on discord to request a new relay path from within Archway Network operators and partners
```
Comment (emperorjm) [inline README.md:25]:
```suggestion
- Use "@Mainnet Validator" or "@Relayer" tag on discord to request a new relay path from within Archway Network operators and partners
```

File: ibc_data_schema.json (hunk 3 above)
Comment (shahbazn) [reply ibc_data_schema.json]: on second thought we probably do not need descriptions here as it will be redundant

**Review by emperorjm — APPROVED** [review]

**Pull request merged**